src/Utils/MockBoot.tsx

```diff
@@ -0,0 +1,21 @@
+import { Breakpoint } from "@artsy/palette/dist/Theme"
+import { Boot } from "Artsy/Router"
+import React from "react"
+
+export const MockBoot: React.SFC<{ breakpoint?: Breakpoint }> = ({
+  breakpoint,
+  children,
+}) => {
+  return (
+    <Boot
+      initialMatchingMediaQueries={[breakpoint]}
+      context={null as any}
+      user={null as any}
+      relayEnvironment={null as any}
+      resolver={null as any}
+      routes={null as any}
+    >
+      {children}
+    </Boot>
+  )
+}
```

Comment:
I dig it 😄 We can expand as needed with time but i like how for now its very explicitly constrained.